README.md

```diff
@@ -27,115 +31,164 @@ repositories' pull requests by using the GitHub Pull Request Builder plugin. It
+to those two servers.
+
+### Automatically setup Jenkins master and slave(s) using Ansible playbooks
+
+There are two Ansible playbooks: one that sets up a Jenkins master node and
+another one that sets up a Jenkins slave node. Read the
+[Ansible instructions](ansible/README.md) for details on how to execute
 the playbooks.
 
-Note: The "master" playbook may fail due to Jenkins temporary unresponsiveness.
-If you see HTTP request errors, try running it again.
+If you wish to have a single system hosting the entire Jenkins instance, both
+playbooks can be executed in the same system. Execute first the Jenkins master
+playbook, stop Jenkins service (systemctl stop jenkins) and then execute the
+Jenkins slave playbook.
+
+Note: The Jenkins playbooks may fail due to network errors. If you see HTTP
+request errors, try executing them again.
+
 
-### Manual Jenkins steps
+### Manually setup Jenkins via web UI
+
+If you do not have a direct network connection to the system(s), you may
+create a SSH tunnel in your workstation to be able to access the Jenkins
+web UI, which is accessible via HTTPS. Example:
+
+`sudo ssh -o "ProxyCommand ssh <user>@<proxy_server> -W %h:%p" -NL 443:localhost:443 root@<jenkins_master>`
+
+While this connection is alive, you can access then Jenkins web UI at
+https://localhost using a web browser.
 
 #### Create administrative jobs
 
 The administrative jobs will help you set up your Jenkins instance, providing a
-simplified way of creating jobs, credentials and slave nodes.
+simplified way of creating jobs, credentials and slaves.
+
+Execute the single Jenkins job by accessing `https://<jenkins_server>/job/seed_job/build`.
+Update the JOB_DESCRIPTORS_FILES parameter with "jenkins_jobs/create_*.groovy".
+This will prevent the creation of jobs which are executed automatically ("trigger"
+jobs), since they would fail if executed automatically without properly configured
+credentials.
 
-If you've set up a fresh Jenkins instance, it should have a single job,
-available at `https://myjenkins.com/job/seed_job`, which creates jobs based on
-a git repository. When executing it, make sure the parameters are adequate for
-your infrastructure.
+#### Create credentials in Jenkins
 
-If you are on a fresh Jenkins installation or do not yet have GitHub credentials
-configured, execute the seed job with a glob that matches only the
-administrative jobs (e.g. "jenkins_jobs/create_*.groovy"). This will prevent
-"trigger" jobs from being created, since they would fail if executed
-automatically without properly configured credentials.
+The Credentials plugin allows you to store and manage credentials in Jenkins.
+In the case of SSH credentials, the SSH key pair must have been created beforehand.
+The following credentials must be created in Jenkins:
 
-#### Create slaves credentials
+##### Setup SSH credentials used to access Jenkins slaves from Jenkins master in Jenkins
```

Comment:
There are too many "Jenkins" in this phrase. I suggest: "Setup SSH credentials used to access Jenkins slaves from master"